### ISSUE TITLE
Training with front-view car images results not good.

### ISSUE THREAD

**Issue by raagapranitha** [elliottwu/unsup3d#36]
Hi,
Thanks for the paper. I appreciate the awesome work. We are students trying to re-train the Unsup3D model on front view car images. However, the results of the training were not good. We trained with 100 epochs and the validation loss in final epoch is 0.70303. We are definitely sure that we are missing something.

Can you please guide/advise on tweaking any parameters to construct good 3D model for front-view car images ?

Thanks

**Comment by elliottwu** [comment]
Hi,

It is true that reconstruction of cars from frontal views are more challenging with our method, particularly for two reasons: 
1. It is harder to represent the shape of the car using a front-view depth map - the sides are invisible under perspective projection. Essentially depth maps have the disadvantage for modeling objects that appear deeper/longer.
2. The variation of views are larger for frontal viewpoints, and only few symmetric counterparts would appear in the same image. This makes matching and alignment more challenging. 

It might be worth checking out a few more recent papers in this regard: https://arxiv.org/abs/2011.00844, https://arxiv.org/abs/2010.09125.
